Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env groovy
+// vim: set ts=4 sts=4 sw=4 noet:
+
+pipeline {
+	agent any
+	environment {
+		DOCKER_IMG = "docker.lco.global/pupe-pat"
+		GIT_COMMIT_SHORT_ID = getCommit().take(12)
+		GIT_TAG_NAME = gitTagName()
+	}
+	stages {
+		stage('Build Docker image') {
+			steps {
+				script {
+					dockerImage = docker.build("${DOCKER_IMG}:${GIT_COMMIT_SHORT_ID}", "--pull .")
+				}
+			}
+		}
+		stage('Push Docker image') {
+			steps {
+				script {
+					// always push the Docker image with the git commit version
+					dockerImage.push()
+
+					if (env.BRANCH_NAME == 'master') {
```

Comment:
`gitDescription()` from the shared libraries calls `git describe` which gets the current tag name. If the current tag name doesn't exist, it returns _tag name-commit count past tag-g short hash_ e.g. `1.0.0-23-g9b6929c19`. This is taken care of when creating `DOCKER_IMG` in the comment for [line 9](https://github.com/LCOGT/pupe-pat/pull/10/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R9).